arch/rv32i/src/csr/mcycleh.rs

```diff
@@ -0,0 +1,8 @@
+use kernel::common::registers::register_bitfields;
+
+// myclceh is the higher 32 bits of the number of elapsed cycles
+register_bitfields![u32,
+mcycleh [
```

Comment:
What about moving the `h` versions into the same file as the "l" versions? Seems a little unnecessary to separate them.

Reply:
Agreed and changed.